weekThree/file_manager.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+class FileManager:
+    def __init__(self, file_path):
+        self.file_path = file_path
+
+    def count_lines(self):
+        if not os.path.exists(self.file_path):
+            print("File not found!")
+            return 0
+        with open(self.file_path, 'r', encoding='utf-8') as file:
```

Comment:
You can improve read handling in file operations, e.g., if the file is malformed or not a text file.